protos/sigstore_common.proto

```diff
@@ -60,13 +79,39 @@ message MessageSignature {
         bytes signature = 2;
 }
 
+// LogId captures the identity of a transparency log.
+message LogId {
+        oneof id {
+                // The unique id of the log, represented as the SHA-256 hash
+                // of the log's public key, computed over the DER encoding.
+                // https://www.rfc-editor.org/rfc/rfc6962#section-3.2
+                bytes key_id = 1;
+                // Currently not used but proposed by
+                // https://datatracker.ietf.org/doc/rfc9162/
+                string oid = 2;
+        }
+}
+
 // This message holds a RFC 3161 timestamp.
 message RFC3161SignedTimestamp {
         // Signed timestamp is the DER encoded TimeStampResponse.
         // See https://www.rfc-editor.org/rfc/rfc3161.html#section-2.4.2
         bytes signed_timestamp = 1;
 }
 
+message PublicKey {
+        // Optional key identifier. Format must be agreed upon out of band
+        // by signer and verifier. See PublicKeyIdentifier for more
+        // information.
+        // Implementors MAY provide a mapping function from hint to PublicKey.
+        string key_id = 1;
+        // DER-encoded public key.
+        optional bytes raw_bytes = 2;
+        KeyEncoding key_encoding = 3;
+        SignatureAlgorithm signature_algorithm = 4;
```

Comment:
@znewman01 @haydentherapper @asraa

Added parameters now for key encoding and signature schema. Would this be enough for the current state? PGP is not yet supported by the bundle format.